packages/ui/src/components/vuestic-components/va-button-dropdown/VaButtonDropdown.demo.vue

```diff
@@ -210,9 +210,9 @@ export default {
   },
```

Comment:
We probably want to hide other months by default.

![image](https://user-images.githubusercontent.com/5394573/123787872-013f4980-d8e4-11eb-854b-06cb6ba9ee87.png)